diff --git a/dist/src/commands.d.ts b/dist/src/commands.d.ts
--- a/dist/src/commands.d.ts
+++ b/dist/src/commands.d.ts
@@ -1,10 +1,4 @@
-/**
- * Switch to pause mode (stop synchronization between prosemirror and ytype)
- * @param {import('prosemirror-state').EditorState} state
- * @param {((tr: import('prosemirror-state').Transaction) => void)?} dispatch
- * @returns {boolean}
- */
-export function pauseSync(state: import("prosemirror-state").EditorState, dispatch: ((tr: import("prosemirror-state").Transaction) => void) | null): boolean;
+export const pauseSync: import("prosemirror-state").Command;
 export function configureYProsemirror(opts?: {
     ytype?: YType | null;
     renderer?: Y.AbstractRenderer | null | undefined;
diff --git a/src/commands.js b/src/commands.js
--- a/src/commands.js
+++ b/src/commands.js
@@ -6,8 +6,6 @@
 /**
  * Switch to pause mode (stop synchronization between prosemirror and ytype)
- * @param {import('prosemirror-state').EditorState} state
- * @param {((tr: import('prosemirror-state').Transaction) => void)?} dispatch
- * @returns {boolean}
+ * @type {import('prosemirror-state').Command}
  */
 export function pauseSync (state, dispatch) {
   const pluginState = ySyncPluginKey.getState(state)
