diff --git a/dist/index.cjs b/dist/index.cjs
index ce89be47cebd61a79c6039950f6dcd10df0e3c7e..33557fd950bab552fcc0c01658bb8d4030b17dd2 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -3580,7 +3580,7 @@ handlers.blur = function (view, _event) {
 };
 handlers.beforeinput = function (view, _event) {
   var event = _event;
-  if (chrome && android && event.inputType == "deleteContentBackward") {
+  if ((chrome || gecko) && android && event.inputType == "deleteContentBackward") {
     view.domObserver.flushSoon();
     var domChangeCount = view.input.domChangeCount;
     setTimeout(function () {
diff --git a/dist/index.js b/dist/index.js
index 2f19364145a8bb8c0f44c738d58271c76078e6f9..ec3ccd65b4fa02fddfc4badc8d6e62ad592a6b7b 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -3839,7 +3839,7 @@ handlers.beforeinput = (view, _event) => {
     // is so spotty that I'm still waiting to see where they are going.
     // Very specific hack to deal with backspace sometimes failing on
     // Chrome Android when after an uneditable node.
-    if (chrome && android && event.inputType == "deleteContentBackward") {
+    if ((chrome || gecko) && android && event.inputType == "deleteContentBackward") {
         view.domObserver.flushSoon();
         let { domChangeCount } = view.input;
         setTimeout(() => {
