diff --git a/src/plugins/sync-plugin.js b/src/plugins/sync-plugin.js
index dccfb76918c8c70fe5a5b12372cdabcc9d8c58dd..8c222cf92566ee0f7c798948ece61746603d3cac 100644
--- a/src/plugins/sync-plugin.js
+++ b/src/plugins/sync-plugin.js
@@ -272,8 +272,10 @@ const restoreRelativeSelection = (tr, relSel, binding) => {
         binding.mapping
       )
       if (anchor !== null && head !== null) {
-        const sel = TextSelection.between(tr.doc.resolve(anchor), tr.doc.resolve(head))
-        tr.setSelection(sel)
+        const maxsize = tr.doc.content.size - 1
+        const clampedAnchor = math.min(math.max(anchor, 0), maxsize)
+        const clampedHead = math.min(math.max(head, 0), maxsize)
+        tr.setSelection(TextSelection.between(tr.doc.resolve(clampedAnchor), tr.doc.resolve(clampedHead)))
       }
     }
   }
